docs/01-running-locally.md

```diff
@@ -0,0 +1,48 @@
+# Basic developer run
+
+`sbt run` and visit `localhost:9000` to confirm the service came up correctly.
+
+## Demonstration of null request
+
+Each request has the following format, of zero or more text blocks, and zero or more rule sets to check.
+
+```
```

Comment:
If `jq` is a dependency of these commands, we should specify it above -- but I'd be tempted to remove it for now, on the assumption that devs who want to use it, will.

Reply:
(super minor)